Removed leftover partial sentence + added some punctuation. (#3299)
raspberrypi/documentation: documentation/asciidoc/computers/raspberry-pi-5/introduction.adoc

=== Turning it off and back on again

When you plug your Raspberry Pi into power for the first time, it will automatically turn on and boot into the operating system without having to push the button. 

> If you are running the Raspberry Pi Desktop, you can initiate a clean shutdown by briefly pressing the power button located on the left-hand side of the board, and then releasing it. This will cause a Desktop menu to appear asking whether you want to Shutdown, Reboot, or Logout. You can select whether you want to shutdown or reboot, or alternatively pressing the power button briefly for a second time will initiate a clean shutdown.

Reviewer: ```suggestion
If you run Raspberry Pi Desktop, you can initiate a clean shutdown by briefly pressing the power button. A menu will appear asking whether you want to shutdown, reboot, or logout. Select an option from the menu or press the power button again to initiate a clean shutdown.
```
Suggestion: clean up some passive wording, trim some extra words, keep every sentence short and limited to a single point. Might as well if we're cleaning this up!

.Shutting down your Raspberry Pi 5 using the power button.
image::images/shutdown.jpg[alt="Desktop menu asking whether you want to Shutdown, Reboot, or Logout",width="100%"]

> NOTE: If you are running your Raspberry Pi OS with a Desktop, you can press the power button twice in quick succession to shutdown. However, if you are running Raspberry Pi OS Lite in headless mode, pressing the power button a single time will initiate a shutdown.

Reviewer: ```suggestion
NOTE: If you run Raspberry Pi Desktop, you can press the power button twice in quick succession to shutdown. If you run Raspberry Pi OS Lite without a desktop, pressing the power button a single time initiates a shutdown.
```
Suggestion: Use the same introduction of "Raspberry Pi Desktop" as above for consistency and familiarity. Use a similar construction to introduce headless mode, but dispense with the "headless" moniker since some users might not remember the term "headless mode" (but anyone not running a desktop will know that they don't have a desktop).
